process/block/sovereignChainBlock.go

```diff
@@ -1138,6 +1276,25 @@ func (scbp *sovereignChainBlockProcessor) CommitBlock(headerHandler data.HeaderH
 	return nil
 }
 
+func (scbp *sovereignChainBlockProcessor) commitEpochStart(header data.HeaderHandler, body *block.Body) {
+	if header.IsStartOfEpochBlock() {
+		scbp.epochStartTrigger.SetProcessed(header, body)
+		scbp.epochStartTrigger.SetProcessed(header, body)
```

Comment:
Dupplicate

Reply:
Right, fixed in future pr, but deleted here also anyway